src/Security/TwoFactor/BackUpCodesManager.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace App\Security\TwoFactor;
+
+use App\Entity\User;
+use Hashids\Hashids;
+
+class BackUpCodesManager
+{
+    const CODES_COUNT = 20;
+    const SALT = '38c936d565cc1d668ef46ff8f7fe33460814e5cc';
```

Comment:
Must be placed in an environment variable.

Reply:
CODES_COUNT too

Reply:
changed